src/Take.Blip.Builder/Utils/SalesForce/SalesForceClient.cs

```diff
@@ -0,0 +1,156 @@
+﻿using Newtonsoft.Json;
```

Comment:
There's quite a bit of repeated code in this class. Could it be refactored to remove repetition?